src/components/Observations/map/ObservationMapView.tsx

```diff
@@ -97,7 +97,7 @@ export default function ObservationMapView({ observationsResults }: ObservationM
   );
 
   return (
-    <Box display='flex' flexDirection='column'>
+    <Box display='flex' flexDirection='column' flexGrow={1}>
```

Comment:
Charlie I added flexGrow so the map gets full height of view.

Reply:
Thanks